packages/frontend/src/components/profile_pic.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Sport } from "@esp-group-one/types";
+import { useState } from "react";
+import { useEffect } from "react";
+import { Tag } from "./tags";
+
+interface Info {
+  sportName: Sport;
+  selected: boolean;
+  abilityLevel: string;
```

Comment:
I would use our `SportInfo` type, defined [here](https://github.com/ESP-Number-One/MatchyoSports/blob/519dc3ccbd98d1ada7e24e9d11d8efcea305c3e1/packages/types/src/user.ts#L10)... tbf I think we probably need an enum of the ability level